changelog.d/578.bugfix

```diff
@@ -0,0 +1 @@
+Track bot-created admin rooms in the bot's room profile data, not just user-created admin rooms.
```

Comment:
What's the user-facing change there? What bug are we fixing?

Reply:
To reproduce:
- invite hookshot to a room & give it state permissions
- run `!hookshot webhook <name>`
- get invited to an admin room DM by hookshot
- reboot the bridge
- run command in the admin room DM that hookshot created for you

Result: Hookshot will ignore commands in that admin room, because it forgot that it is one.

Reply:
Ah yeah now I remember this. I'd just say "Fixed a case where a bridge-created admin room would stop working on restart"

Reply:
Done :)